src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY_INDEX = -1;
+    private static final int FULL_INDEX = 11;
+    private int[] numbers = new int[STACK_CAPACITY];
 
-    public int total = -1;
+    public int getTotal() {
```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.